lib/classes/class-ajax.php

```diff
@@ -89,6 +90,18 @@ public function request() {
 
       }
 
+
+      /**
+       * Update json key to database.
+       */
```

Comment:
@alimuzzaman please be sure to add a `@author tag` so we know who added the method.